tests/testthat/test-mpi.R

```diff
@@ -0,0 +1,38 @@
+context("mpi")
+
+test_that("mpi_sample() works", {
+  skip_on_cran()
+  skip_if(!nzchar(Sys.getenv("CMDSTANR_RUN_MPI_TESTS")))
```

Comment:
We only run MPI tests if this is set. This is so that we don't add MPI as a dev dependency.